src/main/java/com/enderio/base/common/integrations/EnderIOWrenchCompat.java

```diff
@@ -0,0 +1,22 @@
+package com.enderio.base.common.integrations;
+
+import com.enderio.base.common.blockentity.IWrenchable;
+import com.enderio.base.common.init.EIOItems;
+import com.enderio.base.common.tag.EIOTags;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.event.entity.player.PlayerInteractEvent.RightClickBlock;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class EnderIOWrenchCompat {
+    @SubscribeEvent
+    public static void onRightClickBlock(RightClickBlock event) {
+        ItemStack itemInHand = event.getEntity().getItemInHand(event.getHand());
+        if (itemInHand.is(EIOTags.Items.WRENCH) && !itemInHand.is(EIOItems.YETA_WRENCH.asItem()) && event
+            .getLevel()
+            .getBlockEntity(event.getPos()) instanceof IWrenchable) {
```

Comment:
```suggestion
        // @formatter:off
        if (itemInHand.is(EIOTags.Items.WRENCH)
                && !itemInHand.is(EIOItems.YETA_WRENCH.get())
                && event.getLevel().getBlockEntity(event.getPos()) instanceof IWrenchable) {
            // @formatter:on
```
reformat this code for better readability

Reply:
Ah, the autoformatter was bothering me on that one and I had no idea you could disable it like that, thanks!